udata/harvest/backends/dcat.py

```diff
@@ -49,6 +51,18 @@ class DcatBackend(BaseBackend):
     def initialize(self):
         '''List all datasets for a given ...'''
         fmt = guess_format(self.source.url)
+        # if format can't be guessed from the url
+        # we fallback on the declared Content-Type
+        if not fmt:
+            response = requests.head(self.source.url)
+            mime_type = response.headers.get('Content-Type', '').split(';')[0]
```

Comment:
`.split(';', 1)` ensure you're not splitting useless parts. Headers are so messy…